starlark/unpack.go

```diff
@@ -209,14 +209,24 @@ func UnpackPositionalArgs(fnname string, args Tuple, kwargs []Tuple, min int, va
 		return fmt.Errorf("%s: got %d arguments, want %s%d", fnname, len(args), atmost, max)
 	}
 	for i, arg := range args {
-		if err := unpackOneArg(arg, vars[i]); err != nil {
+		if err := UnpackOneArg(arg, vars[i]); err != nil {
 			return fmt.Errorf("%s: for parameter %d: %s", fnname, i+1, err)
 		}
 	}
 	return nil
 }
 
-func unpackOneArg(v Value, ptr any) error {
+// UnpackOneArg unpacks a single argument, represented as a Value, into
```

Comment:
More concisely:

// UnpackArg unpacks a Value v into the variable pointed to by ptr.
// See [UnpackArgs] for details, including which types of variable are supported.